run_clang_tidy.log

```diff
@@ -0,0 +1,2832 @@
+Enabled checks:
+    modernize-return-braced-init-list
```

Comment:
this file should probably not be committed

Reply:
@sawenzel indeed. will remove that one and the one above when updating the PR if / when we decide to go with (and_ when the fair logger related PRs are done, as they created some conflict)